### DISCUSSION TITLE
How to capture event based replication task status changes

### DISCUSSION THREAD

**Discussion by h2cone** [goharbor/harbor#19853] [Q&A] [answered]
I have added event-based replication rules in the specified project, where users trigger my application to push images to that project. Is it possible to programmatically capture the executed tasks, including image names and status changes, without periodically iterating through the entire replication task list?

**Comment by chlins** [comment, marked as answer]
You can configure a webhook policy for your project and subscribe to the replication event. When the event occurs, a notification will be sent to your webhook server.
